CONTRIBUTING.rst

```diff
@@ -247,3 +247,28 @@ exposed to the front end, to properly generate translation files. See error
 message handling in ``metashare/api/sf_run_flow.py`` for an example.
 
 .. _user language is auto-detected at runtime: https://github.com/i18next/i18next-browser-languageDetector
+
+Setting up the GitHub App
+-------------------------
+
+To deploy this app, you will need to set up a GitHub app and give it
+proper permissions. You can do that at
+``https://github.com/organizations/<your org>/settings/apps``
+
+The app will need the following permissions:
+
+ - Repository permissions
+    - Contents: read & write (maybe could be just read?)
+    - Metadata: read-only
+    - Pull requests: read & write (maybe could be just read?)
+    - Commit statuses: read & write
+ - Organization permissions:
+    - None
+ - User permissions:
+    - None
```

Comment:
Needs read for email address, no?

Reply:
I recall setting that, but I can't see that in out set-up app. Let me look for it more.

Reply:
Oh ugh it is there, it's just "Read-only" is the same width as "No access" and so my eyes kept glossing over it.